sdk/appconfiguration/app-configuration/rollup.base.config.js

```diff
@@ -40,9 +40,14 @@ export function nodeConfig(test = false) {
     // different output file
     baseConfig.output.file = "test-dist/index.node.js";
 
-    // mark assert as external
+    // mark assert packages we use as external
     baseConfig.external.push("assert");
 
+    baseConfig.external.push(
+      ...Object.keys(pkg.dependencies),
```

Comment:
Sorry, deps are already included in line 9, so only dev dependencies need to be added here.

Reply:
Fixed.